assets/js/modules/analytics/common/profile-select.test.js

```diff
@@ -38,28 +38,30 @@ apiFetchMock.mockImplementation( ( ...args ) => {
 
 const setupRegistry = ( { dispatch } ) => {
 	const { id, webPropertyId, accountId } = fixtures.propertiesProfiles.profiles[ 0 ];
+	const propertyID = webPropertyId;
```

Comment:
It feels a bit strange to re-assign that here, as that line itself has no context on why it's reassigned. I'd prefer if we could pass `{ propertyID: webPropertyId }` below instead (that's how we do that elsewhere too).

This also applies to other similar re-assignments below - let's rather pass the values within the objects as applicable, since then there is more clear context on why this "rename" / "reassignment" is necessary (because of an object expecting specific properties).

Reply:
Fixed